src/Error/Debugger.php

```diff
@@ -918,10 +918,29 @@ public static function addFormat(string $format, array $strings): array
         } else {
             $self->_templates[$format] = $strings;
         }
+        unset($self->renderers[$format]);
 
         return $self->_templates[$format];
     }
 
+    /**
+     * Add a renderer to the current instance.
+     *
+     * @param string $name The alias for the the renderer.
+     * @param class-string $class The classname of the renderer to use.
```

Comment:
If you know the base type, you can use `class-string<\Cake\My\Type>` to be more specific for static analyzers.